src/server/users/controllers.js

```diff
@@ -69,5 +71,34 @@ export default {
     }).catch((err) => {
       return cb(new APIError(err, {statusCode: err.statusCode, message: err.message}));
     });
+  },
+  addCreatedPost(req, res, cb) {
+    const currentTime = new Date();
```

Comment:
KeyUtils.genTimeHash() need a parameter, Time object.
cause in item controller, we make Time object to make createdDate field.
timehash's time should be same with this createdDate's time so we couldn't make Time object in KeyUtils function.